.github/workflows/pull-request.yaml

```diff
@@ -6,6 +6,10 @@ on:
     paths-ignore:
       - "README.md"
       - "catalog.yaml"
+      - ".github/**"
```

Comment:
So if this workflow is modified, it won't actually run when the PR is submitted.  Is this what you were looking for?  If so, how do we test this workflow?

Reply:
Yeah that makes sense 😅. I'll remove that path ignore from the list.